### PR TITLE
add user days 2024

### PR DESCRIPTION
**Pull request by stephprince**
draft of user days 2024 webpage

### CODE REVIEW

File: HCK20_2024_Remote/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- **Dates:** June 12-14, 2024
+- **Location:** Virtual
+
+## Registration
+
+Register for the event [here](https://forms.gle/pTsjRMEqsgCRo3VD8).
+
+## Logistics
+
+We will be using the Zoom for the meeting. We will send an email in the days before the workshop with Zoom links.
```
Comment (rly) [inline HCK20_2024_Remote/README.md:28]:
```suggestion
We will be using Zoom for the meeting. We will send an email in the days before the workshop with Zoom links.
```

File: HCK20_2024_Remote/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<img alt="NWB 2024 User/Developer Days Banner" src="images/logo_brain_text_white_hor.png">
+
+# NWB User Days 2024
```
Comment (rly) [inline HCK20_2024_Remote/README.md:3]: I know we have a long history of calling this event the "NWB User Days" but this does not really convey what the workshop is about, especially now that it is centered around converting data to NWB. What about "NWB Data Conversion Workshop" or "NWB User Days 2024: Convert Your Own Data"? @stephprince @oruebel @bendichter

Comment (oruebel) [reply HCK20_2024_Remote/README.md:3]: How about including GUIDE in the subheading? E.g. "NWB User Days: A  guide to NWB conversion and adoption"

**Comment by oruebel** [comment]
Looks good to me.

File: HCK20_2024_Remote/README.md (hunk 2 above)
Comment (bendichter) [reply HCK20_2024_Remote/README.md]: I'm in support of changing the name. I like: "NWB Data Conversion Workshop"

File: README.md [hunk 3]
```diff
@@ -7,6 +7,7 @@
 - [Cosyne: February 2024](Cosyne_2024/README.md) - A tutorial session on NWB and DANDI will be held during the
   Computational and Systems Neuroscience (COSYNE) 2024 conference in Lisbon, Portugal on **February 29, 2024**.
 - [NWB Developer Hackathon: April 2024](HCK18_2024_Dev_Hackathon_DataJoint/README.md): **April 17 - 19, 2023** at DataJoint Headquarters in Houston, TX, USA
+- [NWB User Days: June 2024](HCK20_2024_Remote/README.md) - **June 12 - 14, 2024** (virtual)
```
Comment (bendichter) [inline README.md:10]: I'd rather have the numbers in chronological order. Could you please fix the numbering so that this is 19 and the NDRH event is 20?

**Comment by bendichter** [comment]
See #273

File: HCK19_2024_Remote/agenda/AGENDA.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+**The agenda is a work in 
+progress! Further details will be announced here as they become available**
+
+### Agenda
```
Comment (bendichter) [inline HCK19_2024_Remote/agenda/AGENDA.md:4]:
```suggestion
```

fix 
image

**Review by bendichter — APPROVED** [review]

**Pull request merged**